README.md

```diff
@@ -45,56 +23,145 @@ aws_secret_access_key=xxxxxxxx
+aws_access_key_id=xxxxxxxxxx
+aws_secret_access_key=xxxxxxxx
 ```
 
-### Manifest formats
-
-Both the `sync manifest` and `redaction manifest` have the same format as described below.
+*For GCP*
+- Create `gcloud-creds-secret` file containing GCP key
 ```
-id	file_name	md5	size	state	project_id	baseid	version	release	acl	type	deletereason	url
-ada53c3d-16ff-4e2a-8646-7cf78baf7aff	ada53c3d-16ff-4e2a-8646-7cf78baf7aff.vcf.gz	ff53a02d67fd28fcf5b8cd609cf51b06	137476	released	TCGA-LGG	6230cd6d-8610-4db4-94f4-2f87592c949b	1	12.0	[u'phs000xxx']	active		s3://xxxx/ada53c3d-16ff-4e2a-8646-7cf78baf7aff
-2127dca2-e6b7-4e8a-859d-0d7093bd91e6	2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz	c4deccb43f5682cffe8f56c97d602d08	136553	released	TCGA-LGG	6a84714f-2523-4424-b9cb-c21e8177cd0f	1	12.0	[u'phs000xxx']	active		s3://xxxx/2127dca2-e6b7-4e8a-859d-0d7093bd91e6/2127dca2-e6b7-4e8a-859d-0d7093bd91e6.vep.vcf.gz
+{
+  "type": "service_account",
+  "project_id": "dcf-prod",
+  "private_key_id": xxxxxxxx,
+  "private_key": xxxxxxxx,
+  "client_email": "data-replication-dcf-prod@dcf-prod.iam.gserviceaccount.com",
+  "client_id": xxxxxxxx,
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": xxxxxxxx,
+  "client_x509_cert_url": xxxxxxxx
+}
 ```
 
-The `ignored_files_manifest.csv`
+### NIH Token
+- Download token
+Login to NIH (https://portal.gdc.cancer.gov/) and download token under user profile options
+*From root directory*
+- Replace token in config files
+`cd dcfprod/apis_configs/dcf_dataservice`
+`vim dcf_dataservice_settings`
+`vim creds.json`
 
-```
-gcs_object_size	gcs_object_time_stamp	md5sum	acl	gcs_object_url	gdc_uuid
-19490957	2016-03-28T21:51:54Z	bd7b6da28d89c922ea26d145aef5387e	phs000xxx	gs://5aaxxxxxxxx/tcga/PAAD/DNA/AMPLICON/BI/ILLUMINA/C1748.TCGA-LB-A9Q5-01A-11D-A397-08.1.bam	e5c9f17d-8f66-4089-ac70-2a5cde55450f
-```
+- Run job to set secrets for cloud and NIH tokens
+`bash ./dcf-dataservice/jobs/kube-setup-scripts.sh`
+
+## Run Replication Job
+### Dry Run
+*AWS*
+`gen3 runjob aws-bucket-replicate-job RELEASE DR<release number> QUICK_TEST True GDC_BUCKET_NAME gdcbackup MANIFEST_S3 s3://data-refresh-manifest/<active manifest file name> THREAD_NUM 20 LOG_BUCKET data-refresh-output CHUNK_SIZE 1`
+
+*GCP*
+`gen3 runjob google-bucket-replicate-job PROJECT dcf-prod MAX_WORKERS 80 RELEASE DR<release number>  MANIFEST_FILE gs://replication-input/<active manifest file name> IGNORED_FILE gs://replication-input/ignored_files_manifest.csv LOG_BUCKET datarefresh-log`
+
+### If pass
+Replace True with False in previous cmd
 
+## Redaction
+Upload redaction manifest to s3 and run job
+
+### Dry Run
+`gen3 runjob remove-objects-from-clouds-job DRY_RUN True RELEASE DR<release number> MANIFEST_S3 s3://data-refresh-manifest/<obsolete manifest file name> LOG_BUCKET data-refresh-output IGNORED_FILE_S3 s3://data-refresh-manifest/ignored_files_manifest.csv`
+
+### If pass
```

Comment:
dry run to check if anything needs to be redacted. If dry run finds nothing, no need to run the real thing. The job fails if yuou run when you have nothing to redact